spec/linter-eslint-spec.js

```diff
@@ -50,13 +50,12 @@ function copyFileToDir(fileToCopyPath, destinationDir) {
  */
 // eslint-disable-next-line import/prefer-default-export
 export function copyFileToTempDir(fileToCopyPath) {
```

Comment:
Depending on discussion, mark this as `async`.

Reply:
Looking at this change, seems you are already thinking along same lines as what I said in response to the request for deleting the comment. So I will 🔥 the comment and mark this func as `async` and update the pull.

Reply:
done